source/speech/commands.py

```diff
@@ -80,7 +80,7 @@ def _getFormattedDevInfo(self):
 	def __repr__(self):
 		return (
 			f"CancellableSpeech ("
-			f"{ 'cancelled' if self._checkIfCancelled() else 'still valid' }"
+			f"{ 'cancelled' if self._checkIfCancelled() else 'still valid'}"
```

Comment:
@nvdaes - this will be added until #16866 is merged

Reply:
OK, then I'll revert e1d6241